### PR TITLE
feat: Created questions page

### PR DESCRIPTION
**Pull request by arjunbharti**
Created questions page for quizie

- Attempt quiz
- Select from options of four
- Submit button

Screenshots: 
Screenshot from 2022-02-18 17-02-18
Screenshot from 2022-02-18 17-01-38

### CODE REVIEW

File: styles/questions.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    padding: 2rem 20rem;
+}
+
+.questions{
+    padding-top: 1rem;
+}
+
+.question-text{
+    padding-bottom: 1rem;
+}
```
Comment (sameep-s) [inline styles/questions.css:8]: .questions,
.question-text{
    padding-bottom: 1rem;
}

you can do something like this as they have identical properties or you can use utility casses.

**Review by sameep-s — APPROVED** [review]
Take a look at the comments then you can merge.

File: styles/questions.css (hunk 1 above)
Comment (arjunbharti) [reply styles/questions.css:8]: Updated successfully. Thanks for the suggestion

File: styles/questions.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    padding-top: 1rem;
+}
+
+.question-option{
+    padding-top: 0.5rem;
+    padding-left: 1rem;
+}
+
+.btn-container{
+    padding: 2rem 20rem;
```
Comment (akash-1318) [inline styles/questions.css:16]: Here you are using same padding as .questions-container class so you can create an utility class for that.

**Review by akash-1318 — COMMENTED** [review]
Just one suggestion you are using same padding in two classes at multiple places just create an utility class for that else everything looks cool.

File: styles/questions.css (hunk 2 above)
Comment (arjunbharti) [reply styles/questions.css:16]: I'll create utility classes in the component library and update changes here. Thanks for the suggestion.

**Pull request merged**